mobile/lib/db/upload_locks_db.dart

```diff
@@ -118,6 +127,15 @@ class UploadLocksDB {
                   PRIMARY KEY (${_partsTable.columnObjectKey}, ${_partsTable.columnPartNumber})
                 )
                 ''',
+      '''
+                CREATE TABLE IF NOT EXISTS ${_streamUploadErrorTable.table} (
+                  ${_streamUploadErrorTable.columnID} INTEGER PRIMARY KEY,
```

Comment:
Is there any reason for adding separate id? Why can't we just use uploadedFileID as the primary key?

Reply:
Ok It's fine to remove that.